src/command.c

```diff
@@ -933,15 +947,15 @@ static int dv_convert_ipu_v0(struct device *dev, struct dmp_cmb *cmb,
 		list_add(&cmb_node->list_node, &cmb->cmb_list);
 	}
 	cmd_buf = (uint32_t*)((uint8_t*)(cmb_node->logical) + cmb_node->size);
-	
+
 	cmb_node->size += dv_convert_ipu_v0_fill_cmb(&cmd, cmd_buf, 
 				cmb_node->capacity - cmb_node->size,
 				tex_base_addr, rd_base_addr, wr_base_addr);
 	return 0;
 }
 
 int dv_convert_ipu_command(struct device *dev, struct dmp_cmb *cmb,
-			  struct dmp_dv_kcmd *cmd_info)
+			struct dmp_dv_kcmd *cmd_info)
```

Comment:
misalignment, either break line just after the bracket or align next line just after the bracket